bin/fitbenchmarking

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+"""
+This script is the main entry point into the FitBenchmarking software package.
+For more information on usage type fitbenchmarking --help 
+or for more general information, see the online docs at
+docs.fitbenchmarking.com.
+"""
+
+from __future__ import (absolute_import, division, print_function)
+import argparse
+import glob
+import os
+import sys
+
+from fitbenchmarking.fitting_benchmarking import fitbenchmark_group
+from fitbenchmarking.results_output import save_results_tables
+from fitbenchmarking.utils.options import Options
+
+
+def get_parser():
+    """
+    Creates and returns a parser for the args.
+    """
+
+    epilog = '''Usage Examples:
+
+    $ fitbenchmarking benchmark_problems/*
+    $ fitbenchmarking -o myoptions.ini benchmark_problems/simple_tests \
+benchmark_problems/Muon '''
+
+    parser = argparse.ArgumentParser(
+        prog='FitBenchmarking', add_help=True, epilog=epilog,
+        formatter_class=argparse.RawDescriptionHelpFormatter)
+
+    parser.add_argument('-o', '--options-file',
+                        metavar='OPTIONS_FILE',
+                        default='',
+                        help='The path to a %(prog)s options file')
+    parser.add_argument('problem_sets',
+                        nargs='+',
+                        help='Paths to directories containing problem sets.')
+
+    return parser
+
+
+def run(problem_sets, options_file=''):
+    """
+    Run benchmarking for the problems sets and options file given.
+
+    :param problem_sets: The paths to directories containing problem_sets
+    :type problem_sets: list of str
+    :param options_file: he path to an options file, defaults to ''
+    :type options_file: str, optional
+    """
+
+    # Find the options file
+    current_path = os.path.abspath(os.path.curdir)
+    if options_file != '':
+        # Read custom minimizer options from file
+        options_file = glob.glob(options_file)
+        options = Options(options_file)
+    else:
+        options = Options()
+
+    for sub_dir in problem_sets:
+        # generate group label/name used for problem set
+        label = sub_dir.replace('/', '_')
```

Comment:
Will this `fitbenchmarking benchmark_problems/simple_tests benchmark_problems/Muon` create label names `benchmark_problems_simple_tests` and `benchmark_problems_Muon`? We should try and think of a way to keep the original names `simple_tests` and `Muon`

Reply:
For the scope of this issue I think we should leave it as is. @tyronerees suggested having an optional META.txt or similar in each problem set which could give info including the label. Otherwise default to the uderscores as is currently done. I'll create an issue this afternoon to address it.

Reply:
see #345